modules/nf-core/tsebra/main.nf

```diff
@@ -0,0 +1,61 @@
+process TSEBRA {
+    tag "$meta.id"
+    label 'process_single'
+
+    conda "${moduleDir}/environment.yml"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/tsebra:1.1.2.4--pyhca03a8a_0':
+        'biocontainers/tsebra:1.1.2.4--pyhca03a8a_0' }"
+
+    input:
+    tuple val(meta), path(gtfs)
+    path hints_files
+    path keep_gtfs
```

Comment:
Is this a path or a value?

Reply:
`keep_gtfs` is a list of paths similar to `hints_files`.  It starts with a verb and can be confusing. But that's the keyword `keep_gtf` used by tesbra in its code and docs:

https://github.com/Gaius-Augustus/TSEBRA/blob/c87ba3a82df40953d2a9bc19f770b676ad389949/bin/tsebra.py#L173

Reply:
Hmm, still not very clear here though. 
So there is a gtf in the main input as well as this keep_gtfs?

Reply:
I'll try to elaborate further.

- `gtfs`: GTFs containing proposed annotations. If an annotation does not pass the evaluation process, it is dropped.
- `hints_files`: GFF files containing external evidence such as intron and cds hints to evaluate the proposed annotations.
- `keep_gtfs`:  GTFs containing established annotations. These annotations are included in the evaluation process but not excluded.

Reply:
Ok, fine.